srcpkgs/modprobed-db/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'modprobed-db'
+pkgname=modprobed-db
+version=2.44
+revision=1
+build_style=gnu-makefile
+depends="kmod"
+short_desc="Keeps track of EVERY kernel module ever used"
+maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
+license="MIT"
+homepage="https://github.com/graysky2/modprobed-db"
+distfiles="https://github.com/graysky2/${pkgname}/archive/v${version}.tar.gz"
```

Comment:
Please don't use only `pkgname` in the URL. Either use `homepage` or spell it out completely.